src/style.css

```diff
@@ -1 +1,68 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: #3d4e61;
+  box-sizing: border-box;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
```

Comment:
In this block of code, you are applying a top margin to the '.search-bar' class. It would be better to be consistent with your margins and add them only to the top or bottom, but not both, to avoid potential margin collapse (from the checklist).